.editorconfig

```diff
@@ -0,0 +1,34 @@
+# http://editorconfig.org
+root = true
```

Comment:
there is a PR https://github.com/futantan/OpenGpt/pull/10 which is about the format and eslint, please remove this file and we will use that instead, or add this to your .gitignore file

Reply:
done